packages/react-sdk/src/css/units.ts

```diff
@@ -29,4 +29,5 @@ export const units = [
   "vw",
   "x",
   "%",
+  "number",
```

Comment:
file was generated, its even in the comment above

Reply:
I added export const Unit = z.union([z.enum(units), z.literal("number")]); in css/schema number as a valid one, but that's kinda wrong, these two are now out of sync